Assets/Scripts/Control&Input/PlayerInputManagerController.cs

```diff
@@ -86,8 +86,9 @@ private void OnPlayerJoined(PlayerInput playerInput)
         var inputManager = playerInput.GetComponent<InputManager>();
         inputManager.PlayerCamera.enabled = false;
         LocalPlayerInputs.Add(inputManager);
+        if (playerInput.currentControlScheme == "MouseAndKeyboard")
+           inputManager.SetMouseAndKeyboard();
```

Comment:
Couldn't we just put this check into the Start method for InputManager, since it sits on the playerinput and already accesses it, and thus avoid exposing the m&k bool?

Reply:
Nope, unfortunately due to the script execution order, this will return false as the add player delegate is invoked before inout manager start is able to do anything